code/src/components/ToDoList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { TodoItem } from "./ToDoItem.js";
+import { TodoInput } from "./ToDoInput.js";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+
+export const TodoList = () => {
+  const list = useSelector((store) => store.todos.list);
```

Comment:
Since you're only using the items-array, import only that:
`const list = useSelector((store) => store.todos.list.items);`